.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+<!--
+PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.
+
+NOTE: IF YOU HAVE A *QUESTION*, PLEASE POST IT ON THE OKTA DEVELOPER FORUM (https://devforum.okta.com) INSTEAD. ISSUES IN THIS REPOSITORY ARE RESERVED FOR BUG REPORTS AND FEATURE REQUESTS.
+-->
+
+## I'm submitting this issue for the package(s):
+- [ ] jwt-verifier
+- [ ] okta-angular
+- [ ] oidc-middleware
+- [ ] okta-react
+- [ ] okta-react-native
+- [ ] okta-react-native
+- [ ] okta-vue
+
+## I'm submitting a:
+- [ ] Regression (a behavior that used to work and stopped working in a new release)
+- [ ] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
```

Comment:
Bug report & regression can be grouped together as **Bug report**.

This will cut down on the required steps to fill-out this template.